pep-0517.txt

```diff
@@ -166,48 +166,71 @@ The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
 argument is described after the individual hooks::
 
-  def get_build_requires(config_settings):
+  def build_sdist(source_directory, sdist_directory, config_settings):
+      ...
+
+MUST build an unpacked sdist from the ``source_directory`` and place it into the
+specified ``sdist_directory``. If the ``source_directory`` is already an
```

Comment:
An sdist normally contains a top level folder called name-version, e.g. `requests-2.17.3`. Is the backend expected to create this inside `sdist_directory`, or assume that `sdist_directory` is that folder? I.e. which of these is true:

```
from os.path import join, exists
exists(join(sdist_directory, 'requests-2.7.13', 'tests'))
# OR
exists(join(sdist_directory, 'tests'))
```